### PR TITLE
[DNVB-1] Category Page

### PR DESCRIPTION
**Pull request by flair-conor**

### CODE REVIEW

File: assets/scss/addons/_addons.scss [hunk 1]
```diff
@@ -67,12 +67,3 @@
...
 @import "articles";
-
-
-// -----------------------------------------------------------------------------
-// GLOBALS
-// -----------------------------------------------------------------------------
-
-// Settings
-// -----------------------------------------------------------------------------
-@import "settings";
```
Comment (flair-duncan) [inline assets/scss/addons/_addons.scss]: Why have we removed this import?

File: assets/scss/addons/_settings.scss [hunk 2]
```diff
@@ -1,8 +1,72 @@
...
 // TODO - move to an appropriate file
 .button {
     &--primary {
         font-weight: $fontWeight-semibold;
     }
 }
+
+// Max width class for temporary wrapping
+
+@mixin padAlign() {
```
Comment (flair-duncan) [inline assets/scss/addons/_settings.scss]: Why isn't this in functions.scss?

File: assets/scss/addons/_settings.scss [hunk 3]
```diff
@@ -1,8 +1,72 @@
...
+    padding-left: $spacing-triple;
+    padding-right: $spacing-triple;
+}
+
+// Global content heading style
+
+.contentHead {
```
Comment (flair-duncan) [inline assets/scss/addons/_settings.scss]: Why are these definitions in settings.scss?

File: templates/theme-components/category/sidebar.html [hunk 4]
```diff
@@ -17,7 +17,7 @@ <h5 class="sidebarBlock-heading">{{category.name}}</h5>
     {{else}}
         {{> theme-components/category/shop-by-price shop_by_price=category.shop_by_price category_url=category.url}}
     {{/if}}
-</nav> -->
+</nav> --}}
```
Comment (flair-duncan) [inline templates/theme-components/category/sidebar.html]: If we're submitting for PR, please remove commented sections

**Comment by flair-duncan** [comment]
@flair-conor we'll pass this on to @flair-chris now as we said we'll get you started on another project next week

File: assets/scss/addons/_addons.scss (hunk 1 above)
Comment (flair-chris) [reply assets/scss/addons/_addons.scss]: I think this is because it has been imported twice @flair-duncan

Comment (flair-conor) [reply assets/scss/addons/_addons.scss]: @flair-chris Correct. It's imported at the top of the page.

File: assets/scss/addons/_settings.scss (hunk 2 above)
Comment (flair-duncan) [reply assets/scss/addons/_settings.scss]: @flair-chris

File: assets/scss/addons/_settings.scss (hunk 3 above)
Comment (flair-duncan) [reply assets/scss/addons/_settings.scss]: @flair-chris

File: templates/theme-components/category/sidebar.html (hunk 4 above)
Comment (flair-duncan) [reply templates/theme-components/category/sidebar.html]: @flair-chris

File: templates/theme-components/faceted-search/facets/rating.html [hunk 5]
```diff
@@ -34,7 +34,7 @@ <h5 class="accordion-title">
                         </span>
                         {{lang 'search.faceted.rating.and-up'}}
                         {{#if ../show_product_counts}}
-                            <span>({{ count }})</span>
+                            <span>&nbsp;({{ count }})</span>
```
Comment (flair-duncan) [inline templates/theme-components/faceted-search/facets/rating.html]: Please use CSS to position elements

**Comment by flair-duncan** [comment]
Can we please go through the outstanding comments and clear this job off?
- Style definitions in settings
- Remove commented code
Needs a rebase before merging too.

File: templates/theme-components/category/sidebar.html (hunk 4 above)
Comment (flair-chris) [reply templates/theme-components/category/sidebar.html]: resolved in 83ba5d8

File: assets/scss/addons/_settings.scss (hunk 3 above)
Comment (flair-chris) [reply assets/scss/addons/_settings.scss]: resolved in 2412eb0

File: templates/theme-components/faceted-search/facets/rating.html (hunk 5 above)
Comment (flair-chris) [reply templates/theme-components/faceted-search/facets/rating.html]: 63fd350 had to use lang as css margin or padding breaks the underline

**Comment by flair-duncan** [comment]
Needs another rebase

Sort by doesn't look correct, the label has disappeared:
image

Need some spacing between columns when the image is full container width:
image

The qv/compare icons need to appear when the entire section is moused over. I believe Conor set this up, it seems to have reverted:
image

Too much space after the pagination:
image

**Comment by flair-conor** [comment]
> Need some spacing between columns when the image is full container width:
> image

Note: Probably best to put said spacing in the content's section to maintain the aspect ratio that's been set up for the image.

**Pull request merged**